lib/elixir_internal_certificate/scraper/schemas/search_result.ex

```diff
@@ -1,21 +1,40 @@
 defmodule ElixirInternalCertificate.Scraper.Schemas.SearchResult do
   use Ecto.Schema
 
+  import Ecto.Changeset, only: [cast: 3, validate_required: 2, validate_number: 3]
+
+  alias ElixirInternalCertificate.Scraper.Schemas.UserSearch
+
   schema "search_results" do
     field :top_ad_words_total, :integer
     field :ad_words_total, :integer
     field :non_ad_words_total, :integer
     field :links_total, :integer
     field :html_response, :string
+    field :non_ad_words_links, {:array, :string}
+    field :top_ad_words_links, {:array, :string}
 
     belongs_to :user_search,
-               ElixirInternalCertificate.Scraper.Schemas.UserSearch,
+               UserSearch,
                foreign_key: :user_search_id
 
-    has_many :url_results,
-             ElixirInternalCertificate.Scraper.Schemas.UrlResult,
-             foreign_key: :search_result_id
-
     timestamps()
   end
+
+  def create_changeset(search_result, attrs),
+    do:
+      search_result
```

Comment:
Please this [comment](https://github.com/nvminhtue/elixir_internal_certificate/pull/35#discussion_r909340471), it took multiple lines in the body

Reply:
Fixed in eb41b08